openghg/dataobjects/_datamanager.py

```diff
@@ -66,6 +74,11 @@ def refresh(self) -> None:
 
         uuids = list(self.metadata.keys())
         res = search(uuid=uuids)
+        # We don't want the object store in this metadata as we want it to be the
+        # unadulterated metadata to properly reflect what's stored.
+        for m in res.metadata.values():
+            del m["object_store"]
```

Comment:
Would this raise an error if the "object_store" key isn't in this metadata?

Reply:
OK I've fixed this by creating a cleaning function within the `DataManager` class.